src/util.cpp

```diff
@@ -445,6 +445,17 @@ void ArgsManager::ParseParameters(int argc, const char* const argv[])
             m_override_args[key].push_back(val);
         }
     }
+
+    // we do not allow -includeconf from command line, so we clear it here
+    auto it = m_override_args.find("-includeconf");
+    if (it != m_override_args.end()) {
+        if (it->second.size() > 0) {
+            for (const auto& ic : it->second) {
+                printf("warning: -includeconf cannot be used from commandline; ignoring -includeconf=%s\n", ic.c_str());
```

Comment:
`fprintf(stderr, ...`?

Reply:
Makes sense. Done.

Reply:
Actually, I remember now. The reason was because travis fails for certain tests when using fprintf.

Reply:
Should probably fix the test instead in that case :-)

Reply:
I thought it was a travis internal thing to automatically assume error when stderr was printed to. I can't find any reference to this though... digging.

Reply:
It's actually `test_runner.py`, not travis. Digging more.